backend/src/main/java/clf/integra/backend/service/UserService.java

```diff
@@ -1,12 +1,45 @@
 package clf.integra.backend.service;
 
+import clf.integra.backend.model.User;
 import clf.integra.backend.repository.UserRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
 @Service
 @RequiredArgsConstructor
 public class UserService {
 
     private final UserRepository userRepository;
+
+    public double collectTaxesAndFeesFromBranch(UUID branchId) {
+        // We will be able to use the method of get clients for a branch implemented by Madalina
+        // after merging the develop branch with the branch she created.
+        List<User> allUsers = userRepository.getAllUsers();
+        List<User> usersBranch = new ArrayList<>();
+        double revenue = 0;
+
+        for (User user: allUsers) {
+            if (branchId.equals(user.getId())) {
```

Comment:
One thing to take care of -> if the branchId comes as null, it will fail big time, please validate this as well